frappe/utils/__init__.py

```diff
@@ -1166,3 +1168,18 @@ def run(self):
 
 	def reset(self):
 		self._functions.clear()
+
+
+class Truthy:
+	def __init__(self, value=True, context=UNSET):
+		self.value = value
+		self.context = context
+
+	def __bool__(self):
+		return True
+
```

Comment:
> == doesn't work.

I'm assuming we want to implement this for the sake of backwards compatibility where someone is using `doc.has_value_changed("field") == True`. Otherwise, there would be no reason to support this, eh?

Reply:
yeah